buddyboss/buddyboss-platform
# PROD-733 Fix - Bullet list formatted reply from discussion is not showing properly on activity feed

### All Submissions:


Fixes #2993 

### How to test the changes in this Pull Request:

1. Go to the discussion
2. Create a new reply with bullet listed items
3. Submit it
4. See the post in the activity feed

### Proof Screenshots or Video
https://prnt.sc/1uunq68

### Other information:
